### PR TITLE
DIV-5170: Update sign out page to have dynamic dates

### PR DESCRIPTION
**Pull request by Cotterale**
+ Update case-mock.json to have these new dates

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /* eslint-disable max-len */
 const modulePath = 'steps/petitioner/exit-no-longer-wants-to-proceed/ExitNoLongerWantsToProceed.step';
 
-const Exit = require(modulePath);
```
Comment (hosainnet) [inline test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js]: Just curious as to how this ever worked?

Comment (hosainnet) [reply test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js]: Oh nevermind, it's still pointing to the same step, so just a variable rename, ignore me sorry

File: test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js [hunk 2]
```diff
@@ -15,6 +15,25 @@ describe(modulePath, () => {
...
+    const ignoreContent = ['continue', 'serviceName', 'backLink', 'signOut'];
+
+    it('displays issue date', () => {
+      const session = {
+        case: {
+          state: 'AwaitingPronouncement',
+          data: {
+            divorceWho: 'blerp',
+            dateRespondentEligibleForDa: ['2019-09-24T00:00:00.000Z'],
+            dateCaseNoLongerEligibleForDa: ['2020-05-11T00:00:00.000Z']
```
Comment (hosainnet) [inline test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js]: Those dates are not required for this particular test to pass, as it's mainly checking that all content values are present in the template. I think you'd want another test which verifies formatted date is present in the rendered template (not sure how from the top of my head)

File: test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js [hunk 3]
```diff
@@ -15,6 +15,25 @@ describe(modulePath, () => {
...
+
+  describe('Sign out page when user selects no', () => {
+    const ignoreContent = ['continue', 'serviceName', 'backLink', 'signOut'];
+
+    it('displays issue date', () => {
+      const session = {
+        case: {
+          state: 'AwaitingPronouncement',
+          data: {
+            divorceWho: 'blerp',
```
Comment (johnoneill46) [inline test/unit/steps/petitioner/exitNoLongerWantsToProceed.test.js]: also change 'blerp' to husband or wife

**Comment by nsingamsetty** [comment]
This is already tested .Merging now.

**Pull request merged**